.github/workflows/release.yaml

```diff
@@ -28,13 +28,13 @@ jobs:
         with:
             toolchain: "1.62"
             command: build
-            args: --all --release --target=x86_64-unknown-linux-gnu
+            args: --all --release --no-default-features --features "kvm,mshv" --target=x86_64-unknown-linux-gnu
```

Comment:
Don't add this commit. It might break binary compatibility for migrations.

Reply:
Good point. PR Updated.